src/main/java/mate/jdbc/model/Driver.java

```diff
@@ -7,6 +7,14 @@ public class Driver {
     private String name;
     private String licenseNumber;
 
+    public Driver() {
+    }
+
+    public Driver(String name, String licenseNumber) {
+        this.name = name;
+        this.licenseNumber = licenseNumber;
+    }
+
     public Driver(Long id, String name, String licenseNumber) {
         this.id = id;
         this.name = name;
```

Comment:
build your logic either on constructors or on setters, 3 overloaded is too much, try to remove at least one of them